docs/input-kafka.asciidoc

```diff
@@ -375,10 +374,11 @@ we haven't seen any partition leadership changes to proactively discover any new
   * Value type is <<string,string>>
   * There is no default value for this setting.
 
-The class name of the partition assignment strategy that the client uses to
-distribute partition ownership amongst consumer instances. Maps to
-the Kafka `partition.assignment.strategy` setting, which defaults to
-`org.apache.kafka.clients.consumer.RangeAssignor`.
+The name of the partition assignment strategy that the client uses to distribute
```

Comment:
Do think that :
value can be one of [range, round_robin, sticky and cooperative_sticky]

could be better?

Reply:
Thanks, I've changed to a bullet list instead.